tests/test_search.py

```diff
@@ -71,10 +71,9 @@ def test_trivial(self):
         qubo = [[1]]
         init = [1]
         tenure = len(init) - 1
-        scale = 1
         timeout = 1
 
-        search = tabu.TabuSearch(qubo, init, tenure, scale, timeout)
+        search = tabu.TabuSearch(qubo, init, tenure, timeout)
```

Comment:
Adding some floating-point python tests would be great. Maybe also include a few of those scaling tests to verify float tabu gets progressively better as bias resolution increases.

Reply:
I've added a floating point python test, but not any of the scaling tests (have not figured out a way to properly prove that float tabu gets better when the bias range gets smaller)

Reply:
Ok, we can add them later.